test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1285,8 +1295,17 @@ class select_ep_transport : public test_ucp_wireup {
         e1->connect(e2, get_ep_params());
         e2->connect(e1, get_ep_params());
 
+        auto tl1 = rma_transport(e1);
```

Comment:
i think it could be useful with SRD so i'd rather keep it

Reply:
so why need this then?
can we replace is_aws by a more generic condition that checks UCT capabilities?
Actually, I would prefer we didn't have "is_aws" function at all.

Reply:
We temporarily need this because SRD is not merged yet, and with this PR, EFA is the only case of IB without RMA.

I'll simply to remove the check and skip the test when no rma are found on both end, let me know if that's what you meant?

Reply:
> I'll simply to remove the check and skip the test when no rma are found on both end, let me know if that's what you meant?

Yes, it is, since that makes the test more generic

Reply:
ok fixed